kerlescan/inventory_service_interface.py

```diff
@@ -67,6 +70,13 @@ def interleave_systems_and_profiles(systems_result, system_profiles_result):
         system_with_profile["system_profile"]["fqdn"] = system["fqdn"]
         system_with_profile["system_profile"]["updated"] = system["updated"]
 
+        # now add tags if any exist for the system.
+        # system_tags_result is a dict keyed on uuid.
+        if system["id"] in system_tags_result and system_tags_result[system["id"]]:
```

Comment:
Is this equivalent?
```python
if system_tags_result.get(system["id"], None)
```

Reply:
Yes, I think it is!  I'll make the change.

Reply:
...and the default return value is None, so no need for ", None"